reverse-proxy.md

```diff
@@ -254,9 +254,8 @@ Of course you need to modify `<your-nc-domain>` to the domain on which you want
 
 <summary>click here to expand</summary>
 
-**Disclaimer:** It might be possible that the config below is not working 100% correctly, yet. Improvements to it are very welcome!
```

Comment:
So this is not needed anymore?

Reply:
I would say the config should work

Reply:
Maybe something like "This config was tested and should normally work on all modern nginx version if you configure it correctly. Improvements to the config are very welcome!"

Reply:
Sounds good!

Reply:
can you please commiot his?

Reply:
done